toolkit/primitives/domain/src/lib.rs

```diff
@@ -726,7 +761,9 @@ pub struct AdaBasedStaking {
 #[cfg(test)]
 mod tests {
 	use super::MainchainAddress;
```

Comment:
it is fine to use this:
```suggestion
	use super::*;
```